doc/source/tune/_tutorials/_faq.rst

```diff
@@ -191,31 +191,32 @@ See the :ref:`tune-autofilled-metrics` section for a glossary.
 How do I set resources?
 ~~~~~~~~~~~~~~~~~~~~~~~
 If you want to allocate specific resources to a trial, you can use the
-``resources_per_trial`` parameter of ``tune.run()``:
+``resources_per_trial`` parameter of ``tune.run()``, to which you can pass
+a dict or a :class:`PlacementGroupFactory <ray.tune.utils.placement_groups.PlacementGroupFactory>` object:
 
 .. code-block:: python
 
     tune.run(
         train_fn,
-        resources_per_trial={
-            "cpu": 2,
-            "gpu": 0.5,
-            "extra_cpu": 2,
-            "extra_gpu": 0
-        })
+        resources_per_trial=tune.PlacementGroupFactory([
+            {"CPU": 2, "GPU": 0.5},
+            {"CPU": 1},
+            {"CPU": 1},
+        ])
```

Comment:
Can you split this example code block into 1. one example with a dict and 2. one example with the placementgroupfactory? the PlacementGroupFactory is a pretty advanced concept